timur/lib/client/jsx/components/model_viewer.jsx

```diff
@@ -34,9 +34,8 @@ const ModelViewer = ({
     Cell: ({value, original}) => (
       <AttributeViewer
         template={template}
-        document={original}
-        value={value}
```

Comment:
Intentional to drop the value arg?

Reply:
Yes, it now gets computed inside AttributeViewer, since we are passing in record + attribute name already, it is redundant (and confusing) to also pass in the value, so to simplify the arguments here I moved the inference inside the AttributeViewer.